### DISCUSSION TITLE
21.2.8

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1310] [General]
## Releases Notes for 21.2.8
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/draw.io-21.2.8-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/draw.io-21.2.8-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/draw.io-universal-21.2.8.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/drawio-amd64-21.2.8.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/drawio-x86_64-21.2.8.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.2.8/drawio-x86_64-21.2.8.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 21.2.8](https://github.com/jgraph/drawio/blob/v21.2.8/ChangeLog). All changes from 21.2.2 to 21.2.8 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.2.8'>21.2.8</a>.</em>

**Comment by patefoniq** [comment]
I can't run the software:
`the procedure entry point discard virtual memory could not be located in the dynamic link library draw.io`

**Reply by davidjgraph** [reply]
21.1.2 is the last version that supports Windows 7 and Windows 8

**Comment by ahmedmohsin** [comment]
It is very good tool it is help me so much. thank you !!
